listenbrainz/webserver/templates/recommendations_cf_recording/info.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'recommendations_cf_recording/base.html' %}
+
+{% block profile_content %}
+
+    <div style="margin-top:20px">
+        <p>
+            The recordings shown in this tab have been generated by a collaborative filtering algorithm using your past listening history. We’re currently experimenting with this technology and are aiming to generate compelling playlists for our users in the near future. We’re sharing these recordings with you in order to get feedback about what you think of these recommended recordings. The recordings fall into two categories and both are generated from the recordings that you’ve listened to in the last week:
```

Comment:
this is loooooong

Reply:
Long? This is the info page, it will be a lil long.